### ISSUE TITLE
Dataset for "Fine-Tuning Pre-trained MAE for classification"

### ISSUE THREAD

**Issue by royleung01** [facebookresearch/mae#169]
HI All,  I am new here to learn MAE ! Good to see you all.

I am trying to run the "Fine-tuning Pre-trained MAE for Classification" Evaluation with ImageNet fine-tuned models.

I follow the instruction to download the "fine-tuned Checkpoint".  However, I cannot give the same/nearest result of the "Reference ImageNet accuracy" (83.664, 85.952, 86.928).  I put ImageNet "Train and Val" dataset to the folder.  

The result in "ViT-Base" as follow. 

python main_finetune.py --eval --resume ../mae_finetuned_vit_base.pth --model vit_base_patch16 --batch_size 16 --data_path ..\ILSVRC\Data\DET --device directml

[00:15:02.980749] Test:  [    0/19299]  eta: 5 days, 23:09:28  loss: 9.4609 (9.4609)  acc1: 0.0000 (0.0000)  acc5: 0.0000 (0.0000)  time: 26.7044  data: 23.3047
[00:15:17.087085] Test:  [   10/19299]  eta: 19:52:32  loss: 8.7052 (8.6560)  acc1: 0.0000 (0.0000)  acc5: 0.0000 (1.7045)  time: 3.7095  data: 2.1197
[00:15:27.845380] Test:  [   20/19299]  eta: 13:08:56  loss: 8.7110 (8.6961)  acc1: 0.0000 (0.0000)  acc5: 0.0000 (1.4881)  time: 1.2429  data: 0.0009
<SNIP>
[05:32:32.324447] Test:  [19298/19299]  eta: 0:00:00  loss: 8.8026 (9.3633)  acc1: 0.0000 (0.0214)  acc5: 0.0000 (0.2254)  time: 0.8783  data: 0.0009
[05:32:35.807836] Test: Total time: 5:17:59 (0.9886 s / it)
[05:32:35.809842] * Acc@1 0.021 Acc@5 0.225 loss 9.363
[05:32:35.814854] Accuracy of the network on the 308782 test images: 0.0%

As my device no Nvidia based machine, i cannot use CUDA but add a torch_directML.   I am no sure if it is affected. 

I would like to ask if i load the right dataset ImageNet ILSVRC2017 DET Dataset "Train and Val" for this Evaluation?
also can i use directML for this program?  

Thanks and much appreciated !

Regards,

Roy Leung

**Comment by JialinWan000** [comment]
Hello, I also encountered the same problem, may I ask if you have solved it now
